### ISSUE TITLE
All instances not created with load_mask() and not visualized with display_instances()

### ISSUE THREAD

**Issue by kecyarchana** [matterport/Mask_RCNN#2755]
Hello,

I am trying to train Mask R-CNN with my own dataset (images .tif, annotations in shapefile) by referring to codes available in this repo. While checking if the masks are correctly loaded or not with the code below I can see only 2 instances for each image, even though there are multiple instances and I can see the properties x, y with train_dataset.image_info[ ].

While looking at the information for a random example train_dataset.image_info[87]. I get the following results
![image_info](https://user-images.githubusercontent.com/22496671/148616935-d638b051-067b-4cf6-b6ab-e6b0b5569ab9.PNG)

But visualizing the image only 2 instances is seen, even though there are around 12.

![87_results](https://user-images.githubusercontent.com/22496671/148616985-e2660751-5fef-4474-8d3d-499d13707800.PNG)

The code for my load_mask is as follows :

 ```
 def load_mask(self, image_id):
        info = self.image_info[image_id]
        num_ids = []
        mask = np.zeros([info['height'], info['width'], len(info['polygons'])], dtype=np.uint8) #Change datatype?
        for i, p in enumerate(info['polygons']):
            # Get indexes of pixels inside the polygon and set them to 1
            rr, cc = skimage.draw.polygon(p['all_points_y'], p['all_points_x'])
            mask[rr, cc, i] = 1
            if p['attribute'] == 'deciduous':
                num_ids.append(1)
            elif p['attribute'] == 'conifers':
                num_ids.append(2)
     
        class_ids = np.array(num_ids, dtype=np.int32)
        return mask.astype(bool), class_ids 
```

The code I am using to look at random example is 

```
image_id = 89
image = train_dataset.load_image(image_id)
mask, class_ids = train_dataset.load_mask(image_id)
bbox = utils.extract_bboxes(mask)
# Load and display
visualize.display_instances(image, bbox, mask, class_ids, train_dataset.class_names)
```

Is there anything I should change in load_mask or visualize.py to visualize /load all the masks for each image? 
Any help would be greatly appreciated. Thankyou!
